jekyll/documentation.html

```diff
@@ -147,6 +147,12 @@ <h3>
         who have never programmed before, or are new to the language.
         </p>
       </div>
+      
```

Comment:
This needs to be in a separate `row`, i.e. you need to put it in `<div class="row">` outside of the current 2nd row.

Reply:
I see, thanks

Reply:
fixed